### DISCUSSION TITLE
Resource in Cluster to actual git location

### DISCUSSION THREAD

**Discussion by dariozachow** [fluxcd/flux2#4810] [Q&A]
Hi, I was wondering if it's possible to get the exact git location (in my case Gitlab) of the Resource applied over flux. Imagine having a configmap in a repository and now I want to know where the file is located at when inspecting the configmap in my cluster.

**Comment by stefanprodan** [comment]
It is not possible to determine the exact location in Git for Kubernetes object, for example a ConfigMap may be generated at build time from some dotenv file, or it may be composed from various Kustomize overlays from different repositories.

**Comment by dariozachow** [comment]
oh, I understand but thanks for your quick response :)
